lib/arena.rb

```diff
@@ -11,41 +11,59 @@ def add_gladiator(glad)
     end
   end
 
+  # sparta and troy are getters for the weapon of the two gladiators currently in the arena
+  # julius and augustus remove the losing gladiator from the arena.
+
   def fight
+    def sparta
+      @gladiators[0].weapon
+    end
+
+    def troy
+      @gladiators[1].weapon
+    end
+
+      def julius
+        @gladiators.delete_at(0)
+      end
+
+      def augustus
+        @gladiators.delete_at(1)
+      end
```

Comment:
Very interesting setup using these methods! My only complaint here is that it's not clear what each method does based on its name.